src/newsletter-editor/store.js

```diff
@@ -77,9 +87,10 @@ export const registerStore = () => register( store );
 
 // Hook to use the retrieval status from any editor component.
 export const useIsRetrieving = () =>
-	useSelect( select =>
-		select( STORE_NAMESPACE ).getIsRetrieving()
-	);
+	useSelect( select => {
+		const { getIsRetrievingData, getIsRetrievingLists, getIsRetrievingSyncErrors } = select( STORE_NAMESPACE );
+		return getIsRetrievingData() || getIsRetrievingLists() || getIsRetrievingSyncErrors();
```

Comment:
Since this is a hotfix, I decided not to remove this hook and maintain its previous behavior so I can avoid making too many changes that aren't related to this issue.